docs/File-System-API.md

```diff
@@ -58,14 +58,69 @@ OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
 FileOutStream out = tfs.getOutputStream(path, options);
 ```
 
+### IO Options
+
+There are two different types of storage in Tachyon, natively managed storage and under storage.
+Natively managed storage is the memory, SSD, and/or HDD which is allocated to Tachyon workers. For
+read and write operations, users can specify the `NativeStorageType` to fine tune the interaction
```

Comment:
users can specify the interaction with Tachyon's native storage through the `NativeStorageType`.